velox/exec/tests/SortBufferTest.cpp

```diff
@@ -397,6 +408,7 @@ TEST_F(SortBufferTest, spill) {
       ASSERT_FALSE(spillStats.has_value());
     } else {
       ASSERT_TRUE(spillStats.has_value());
+      ASSERT_GT(spillStats->spillRuns, 0);
       ASSERT_GT(spillStats->spilledRows, 0);
       ASSERT_LE(spillStats->spilledRows, totalNumInput);
       ASSERT_GT(spillStats->spilledBytes, 0);
```

Comment:
Could you add a test case that verify if spilling is not configured in sort buffer, then spill() call throw? thanks!